### DISCUSSION TITLE
How to List Contacts by a Specific Label via API

### DISCUSSION THREAD

**Discussion by eloimarquessilva** [chatwoot/chatwoot#9233] [Chatwoot APIs]
Hello, I need to list all contacts that are labeled with a specific label via the API, but I can’t find how to do it in the API docs. Is there no endpoint for this?

**Comment by lucianomarcelino** [comment]
https://{{url}}/api/v1/accounts/{{account_id}}/conversations?labels={{label_name}}&status={{status}}&page=0&size=50
